homeassistant/components/daikin/switch.py

```diff
@@ -128,6 +128,7 @@ class DaikinToggleSwitch(SwitchEntity):
     """Switch state."""
 
     _attr_icon = TOGGLE_ICON
+    _attr_name = "Toggle"
```

Comment:
What do we toggle exactly?

Reply:
I mean we are passing a device info, and from what I understand is this switch the main switch of the device, so maybe try to set the name to None
```suggestion
    _attr_name = None
```

This will make it use the device name as entity name